src/Product/ProductRepository.php

```diff
@@ -174,12 +174,12 @@ public function find_delete_product_ids( array $ids, int $limit = - 1, int $offs
 	 * @param int   $limit  Maximum number of results to retrieve or -1 for unlimited.
 	 * @param int   $offset Amount to offset product results.
 	 *
-	 * @return FilteredProductList List of WooCommerce product IDs after filtering.
+	 * @return FilteredProductList List of WooCommerce products after filtering.
 	 */
 	public function find_sync_ready_product_ids( array $args = [], int $limit = - 1, int $offset = 0 ): FilteredProductList {
 		$results = $this->find( $this->get_sync_ready_products_query_args( $args ), $limit, $offset );
 
-		return $this->product_filter->filter_sync_ready_products( $results, true );
+		return $this->product_filter->filter_sync_ready_products( $results );
```

Comment:
💅 

The comment of this function could be updated, so it does not mention that we are returning IDs. 

`Find and return an array of WooCommerce product IDs ready to be submitted to Google Merchant Center.`

Also the name of this function is confusing as `find_sync_ready_product_ids` sounds like will return `productIds` instead of `FilteredProductList`